app/helpers/automated_tests_helper.rb

```diff
@@ -1,12 +1,42 @@
 # Helper methods for Testing Framework forms
 module AutomatedTestsHelper
+  # Create a repository for the test scripts and test support files
+  # if it does not exist
+  def create_test_repo(assignment)
+    # Create the automated test repository
+    unless File.exist?(MarkusConfigurator.markus_config_automated_tests_repository)
+      FileUtils.mkdir(MarkusConfigurator.markus_config_automated_tests_repository)
+    end
+
+    test_dir = File.join(MarkusConfigurator.markus_config_automated_tests_repository, assignment.short_identifier)
+    if !(File.exist?(test_dir))
+      FileUtils.mkdir(test_dir)
+    end
+  end
+
+  def add_test_support_file_link(name, form)
+    link_to_function name do |page|
+      test_support_file = render(:partial => 'test_support_file_upload',
+                                 :locals => {:form => form,
+                                             :test_support_file => TestSupportFile.new})
```

Comment:
Line is too long. [88/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.